pom.xml

```diff
@@ -68,13 +68,14 @@
         <java.level>8</java.level>
         <no-test-jar>false</no-test-jar>
         <groovy-cps.version>1.32</groovy-cps.version>
+        <workflow-step-api.version>2.23-rc567.0fe52fbdf6b5</workflow-step-api.version>
```

Comment:
accidentally  updated `workflow-step-api` before I had `credentials-binding` incremental.  Will update in the next  commit